tests/users/test_models.py

```diff
@@ -662,9 +661,9 @@ def test_get_full_name(self):
         assert len(JobSeekerFactory(first_name=too_long_name, last_name="maréchal").get_full_name()) == 70
 
 
-class JobSeekerProfileModelTest(TestCase):
-    def setUp(self):
-        super().setUp()
+class TestJobSeekerProfileModel:
+    @pytest.fixture(autouse=True)
+    def setup_fixture(self):
```

Comment:
Si tu veux, il y a le hook `setup_method` : https://docs.pytest.org/en/stable/how-to/xunit_setup.html#method-and-function-level-setup-teardown